saleor/plugins/webhook/plugin.py

```diff
@@ -239,6 +239,18 @@ def account_change_email_requested(
             new_email,
         )
 
+    def account_email_changed(
+        self,
+        user: "User",
+        previous_value: None,
```

Comment:
Wrong typing for `previous_value`, it should be rather `Optional["User"]` I think

Reply:
I guess in this case all of the previous_values types needs a refactor here - the almost only types expected are `None` and `Any` but I see previous_value should represent instance or instences type